src/shared/components/hero-bg.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+import React, { ReactNode } from "react";
+
+interface HeroBgProps {
+  children?: ReactNode;
+}
+
+const HeroBg: React.FC<HeroBgProps> = ({ children }) => {
+  return (
+    <div className="relative w-full">
+      <Image
+        className="w-full h-auto bg-cover"
+        src="/hero-bg.png"
+        alt="logo"
+        width={1920}
+        height={1080}
+        layout="w-full h-auto"
```

Comment:
for this image need add attribute "[quality](https://nextjs.org/docs/app/api-reference/components/image#quality)" with 90 or 95 value. This will improve the quality and not lose optimization